draft-zsf.md

```diff
@@ -0,0 +1,176 @@
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200. [2]
+
+The term "Block Rewards" refers to the algorithmic
+issuance of ZEC to every block's creator -- part of the consensus rules.
+
+"Issuance" - The method by which unmined or unissued ZEC is converted to ZEC available to users of the network
+
+"We" - the ZIP authors, owners listed in the above front matter
+
+"`MAX_MONEY`" is the ZEC supply cap. For simplicity, this ZIP defines it to be `21,000,000 ZEC`, although this is slightly larger than the actual supply cap of the original ZEC issuance mechanism.
+
+# Abstract 
+
+This ZIP describes the motivation, the necessary changes for, and the implementation specifications for the Zcash Sustainability Fund (ZSF). The ZSF is a proposed alteration to the block rewards system and accounting of unmined ZEC that allows for other sources of funding besides unissued ZEC. This new mechanism for deposits -- that new applications or protocol designs can use to strengthen the long-term sustainability of the network  -- will likely be an important step for future economic upgrades, such as a transition to Proof of Stake or Zcash Shielded Assets, and other potential protocol fees and user applications.
+
+The changes in this ZIP are ultimately minimal, only requiring for the node to track state in the form of a `ZSF_BALANCE`, and for a new transaction field to be added, called `ZSF_DEPOSIT`. While wallet developer would be encouraged to add the `ZSF_DEPOSIT` field to their UIs, no changes or new behavior are absolutely required for developers or ZEC holders.
+
+This ZIP does not change the current ZEC issuance schedule. Any additional amounts paid into the sustainability fund are reserved for use in future ZIPs.
+
+# Motivation
+
+The Zcash network's operation and development relies fundamentally on the block reward system inherited from Bitcoin. This system currently looks like this:
+
+- At Every New Block:
+    - Miner and funding streams are rewarded a constant amount via unissued ZEC (this constant amount halves at specified heights)
+    - Miner is rewarded via Transaction fees `(inputs - outputs)`
+
+The Zcash Sustainability Fund is a proposed replacement to that payout mechanism, with the relevant parts in *bold* below:
+
+- **Unmined ZEC is now accounted for as `ZSF_BALANCE`**
+- **Transaction includes optional contributions to ZSF via a `ZSF_DEPOSIT` field**
+- Thus, at Every New Block:
+    - Miner and funding streams rewarded the same constant amount, **but from `ZSF_BALANCE`** (this constant amount still halves at specified heights)
+    - Miner is rewarded via Transaction fees `(inputs - outputs)`, **including the `ZSF_DEPOSIT` amount**
+
+This design gives similar clarity and algorithmic control benefits, while also allowing other sources of funds for Block Rewards in addition to newly issued ZEC, via ZSF Deposits.
+
```

Comment:
This is already mentioned in point 1 below.

In general, we discourage specification authors from duplicating the same ideas, because each duplicate needs to be reviewed for consistency. (And if there's a change, each duplicate needs to be changed.)

```suggestion
```